analytics/js_tracking_api.rst

```diff
@@ -732,18 +732,18 @@ Example of usage
         Click me!
     </button>
 
-Setting Download delay
-++++++++++++++++++++++
+Setting link tracking delay
++++++++++++++++++++++++++++
 
-After each outbound link there is a small time frame after which the file will download that will
-ensure there is enough time to track downloads.
-That time frame is set by default to 500ms. To modify it you can use ``setLinkTrackingTimer`` function::
+After each outbound/download link click there is introduced a small delay after which the browser
```

Comment:
I'm not an expert but shouldn't it rather be "... there is a small delay introduced ..." - somehow sounds better to me.